rpi-eeprom-update

```diff
@@ -707,17 +731,18 @@ BOOTLOADER_LATEST=${bootloader_new}
 VL805_CURRENT="${vl805_cur}"
 VL805_LATEST="${vl805_new}"
 VL805_EEPROM="${vl805_eeprom}"
+BOOTLOADER_AUTO_UPDATE_MIN_VERSION=${min_ver}
 EOF
    else
       [ "${HAVE_VL805_EEPROM}" = "0" ] && vl805_eeprom="false" || vl805_eeprom="true"
       cat > "${MACHINE_OUTPUT}" <<EOF
 {
   "EXITCODE": "${exit_code}",
-  "BOOTLOADER_CURRENT": ${bootloader_cur},
```

Comment:
Seems strange to make this change in the `else` branch but not the `if` branch? (was this removed accidentally?)

Reply:
Yes, clearly it was too late in the evening to be writing scripts